ext/afform/admin/ang/afGuiEditor/config-form.html

```diff
@@ -84,5 +84,16 @@
       <p class="help-block">{{:: ts('Choose which contact from the search should match the contact being viewed.') }}</p>
     </div>
   </fieldset>
+  <fieldset>
+
+    <legend>{{:: ts('Submit Actions') }}</legend>
 
+    <div class="form-group" ng-class="{'has-error': !!config_form.redirect.$error.pattern}">
+      <label for="af_config_redirect">
+        {{:: ts('Post-Submit Page') }}
+      </label>
+      <input ng-model="afform.redirect" name="redirect" class="form-control" id="af_config_redirect" title="{{:: ts('Post-Submit Page') }}" pattern="^((http|https):\/\/|\/|civicrm\/)[-0-9a-zA-z\/_.\?=\&\#]+$" title="{{:: ts('Post-Submit Page must be either an absolute url, a relative url or a path starting with CiviCRM') }}"/>
```

Comment:
That looks like a lowercase `z` in `A-z`

Reply:
Maybe we should just do `.+` instead of trying to list every url-safe character I'm sure we missed some (like `%`).

Reply:
@colemanw I think I have resolved this now and also fixed the regex on the server-route (it too had a lower case z in the wrong place)